website/src/main/tut/docs/rules/DisableUnless.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: docs
+title: DisableUnless
+---
+
+# DisableUnless
+
+_Since 0.5.7-67-5b643194-SNAPSHOT_
```

Comment:
should be since 0.5.8 even if that release is not out yet, otherwise we would have to update the docs when 0.5.8 comes out.